samples/app-cache-meetings/nodejs/assets/sample.json

```diff
@@ -9,7 +9,7 @@
       "This sample feature shows how to improve the subsequent loading time of an App that the user has visited during the current meeting. The tab is loaded as an sidepanel in meeting."
```

Comment:
sample.json does not explain about personal tab cache.
Same in Readme file as well.

Reply:
updated in readme file as well